boards/arm/xilinx_kv260_starter_kit_r5/doc/index.rst

```diff
@@ -0,0 +1,79 @@
+.. _xilinx_kv260_starter_kit_r5:
+
+Xilinx KV260 Starter Kit RPU Cortex-R5
+######################################
+
+Overview
+********
+
+This configuration provides support for an ARM Cortex-R5 CPU and these devices:
+
+* ARM PL-390 Generic Interrupt Controller
+* Xilinx Zynq TTC (Cadence TTC)
+* Xilinx Zynq UART
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
+
++--------------+------------+----------------------+
+| Interface    | Controller | Driver/Component     |
++==============+============+======================+
+| GIC          | on-chip    | generic interrupt    |
+|              |            | controller           |
++--------------+------------+----------------------+
+| TTC          | on-chip    | system timer         |
++--------------+------------+----------------------+
+| UART         | on-chip    | serial port          |
++--------------+------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Timer
+------------
+
+This board configuration uses a system timer tick frequency of 1000 Hz.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+on-chip UART0.
+
+Known Problems or Limitations
+==============================
+
+The following platform features are unsupported:
+
+* Dual-redundant Core Lock-step (DCLS) execution is not emulated.
```

Comment:
This is a limitation of the emulator.
Is this really a limitation of the zephyr port?
Can we build lock step and split mode today?  If so how?
If not we can defer to another PR but document which you are doing.

Reply:
Just reworded, it seems a limitation from zephyr port here.